### PR TITLE
Pt 154358145 polish oracles

### PR DESCRIPTION
**Pull request by hanssv**
Some fixes and improvements to oracles:
  *  Rename interaction to query everywhere
  * Fix incorrect deserialize for oracle_register_tx
  *  Remove unused API for now
  * Add a link to aeternity/protocol for Oracle WS API in RELEASE_NOTES
  * Bump DB path in deployment

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # About this release
 
 [This release](https://github.com/aeternity/epoch/releases/tag/v0.4.1) is focused on stability of the testnet - the public test network of nodes and introduces some API for apps usability build on top of it.
-It also introduces a backward incompatible change in the chain format.
+In this release the [Oracle WebSocket API](https://github.com/aeternity/protocol/blob/epoch-v0.4.1/oracles/oracle_ws_api.md) is now documented and can be used to interact with Oracles on the block chain.
```
Comment (lucafavatella) [inline RELEASE-NOTES.md]: feel free to refer to tag epoch-v0.5.0 directly (optional)

Comment (lucafavatella) [inline RELEASE-NOTES.md]: In section "Retrieve the software for running a node" of release notes, close to where the Swagger schema is introduced and linked, could you add a sentence and link to the (versioned) aeternity/protocol repo portion (section / subfolder) meant to document the epoch API?

File: apps/aecore/src/aec_subscribe.erl [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 -type id() :: {ws, ws_handler:id()}.
 -type aeo_event() :: {query, aeo_oracles:id()}
-                   | {response,  aeo_oracles:id(), aeo_interaction:id()}.
+                   | {response, aeo_query:id()}.
```
Comment (lucafavatella) [inline apps/aecore/src/aec_subscribe.erl]: I understand oracle query id includes oracle id.

File: RELEASE-NOTES.md (hunk 1 above)
Comment (hanssv) [reply RELEASE-NOTES.md]: Ok, it was v0.4.1 one line above so I thought better to be consistent!

File: apps/aecore/src/aec_subscribe.erl (hunk 2 above)
Comment (hanssv) [reply apps/aecore/src/aec_subscribe.erl]: Not really, this is just a proof of how utterly useless Dialyzer is at some things. That type was blatantly wrong before, so this is a renaming _and_ a spec-fix. (In fact if you remember the list comprehension you asked me to change yesterday, this was the incorrect type-spec for it...)

File: RELEASE-NOTES.md (hunk 1 above)
Comment (hanssv) [reply RELEASE-NOTES.md]: There is no such repo portion AFAIK?!

Comment (hanssv) [reply RELEASE-NOTES.md]: But now there is :-)

**Comment by hanssv** [comment]
Updated RELEASE_NOTES.md with API links

**Comment by ThomasArts** [comment]
All tests passed

**Pull request merged**